server/src/main/java/com/cloud/configuration/Config.java

```diff
@@ -504,7 +504,7 @@ public enum Config {
             "300",
             "The time interval in seconds when the management server polls for snapshots to be scheduled.",
             null),
-    KVMSnapshotEnabled("Hidden", SnapshotManager.class, Boolean.class, "kvm.snapshot.enabled", "false", "Whether volume snapshot is enabled on running instances on a KVM host", null),
+    KVMSnapshotEnabled("Hidden", SnapshotManager.class, Boolean.class, "kvm.snapshot.enabled", "true", "Whether volume snapshot is enabled on running instances on a KVM host", null),
```

Comment:
@rohityadavcloud for an upgraded environment this default value won't be reflected. either we will need an mysql update statement in the upgrade sql file or move this configuration to ConfigKey object.

Reply:
Okay. I think it is also good to have consistent values and moving to ConfigKey. I've the fix, let me please push it here.